src/ws/activate-motd

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+port=${1:-9090}
+hostname=${2:-$(hostname -f)}
+ip=${3:-$(ip -o route get 255.0 2>/dev/null | sed 's/.*src \([^ ]*\) .*/\1/')}
+
+if test -n "${ip}"; then
+  ip_url=" or https://${ip}:${port}/"
```

Comment:
We don't need the trailing slash (neither below).

Also, four-char indentation, or just `[ -z "$ip" ] || ip_url=...` for brevity.